### PR TITLE
feat: expose log export to users

### PR DESCRIPTION
**Pull request by njhale**
- Add a catch-all modal for unhandled exceptions with download button
that lets users download a zip containing their acorn logs
- Add settings page with export logs option

### CODE REVIEW

**Comment by njhale** [comment]
Addresses https://github.com/gptscript-ai/desktop/issues/84

File: app/settings/page.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    }, 2000);
+  };
+
+  return (
+    <div className="container mx-auto max-w-6xl py-10 px-6">
+      <h1 className="text-3xl font-bold mb-8">Settings</h1>
+      <Divider className="my-6 w-full" />
+      <div className="space-y-8">
+        <section>
+          <h2 className="text-2xl font-semibold mb-4">Trouble Shooting</h2>
```
Comment (g-linville) [inline app/settings/page.tsx:25]: nit: I normally see this spelled as one word

```suggestion
          <h2 className="text-2xl font-semibold mb-4">Troubleshooting</h2>
```

File: app/settings/page.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+export default function Settings() {
+  const [loading, setLoading] = useState(false);
+
+  const handleDownload = () => {
+    setLoading(true);
+    // Simulate download delay
+    setTimeout(() => {
+      setLoading(false);
+      window.location.href = '/api/logs';
+    }, 2000);
```
Comment (g-linville) [inline app/settings/page.tsx:16]: Was this just for testing or do we want it to behave this way?

**Review by g-linville — APPROVED** [review]

File: app/settings/page.tsx (hunk 2 above)
Comment (StrongMonkey) [reply app/settings/page.tsx:16]: I know all nick wants is that spinner :)

Comment (njhale) [reply app/settings/page.tsx:16]: Ah yeah, meant to drop this, nice catch

File: components/navbar/me.tsx [hunk 3]
```diff
@@ -9,12 +9,13 @@ import {
...
 } from '@nextui-org/react';
 import { useState, useEffect, useContext } from 'react';
-import { GoPersonFill } from 'react-icons/go';
+import { GoPersonFill, GoGear } from 'react-icons/go';
 import Logout from '@/components/navbar/me/logout';
 import { AuthContext } from '@/contexts/auth';
 import { loginThroughTool } from '@/actions/auth/auth';
 import { getMe } from '@/actions/me/me';
 import Loading from '@/components/loading';
+import Link from 'next/link';
```
Comment (ryanhopperlowe) [inline components/navbar/me.tsx:18]: Nit: Can we keep consistency and use `NextLink` throughout the app?

**Review by ryanhopperlowe — APPROVED** [review]

File: components/navbar/me.tsx (hunk 3 above)
Comment (njhale) [reply components/navbar/me.tsx]: done!

**Pull request merged**